x/ccv/provider/keeper/msg_server.go

```diff
@@ -469,25 +469,12 @@ func (k msgServer) RemoveConsumer(goCtx context.Context, msg *types.MsgRemoveCon
 
 	phase := k.Keeper.GetConsumerPhase(ctx, consumerId)
 	if phase != types.ConsumerPhase_CONSUMER_PHASE_LAUNCHED {
```

Comment:
Why only stop launched consumers? This means that there is no way to stop and delete a consumer that is just registered. We should be able to stop and delete consumers that are not yet launched without waiting to the unbonding period.

Reply:
Discussed offline. Decided to "Ignore for now. Low priority"

My point of view:
> This adds additional complexity (check on whether we can delete everything now or later) in my view for no real benefit. Additionally, if you haven't started yet, there's probably not much to delete to start with, so no reason to start a deletion. Also, I do not think conceptually it makes sense to remove a registered chain. If you don't like it, just leave it as is and later re-register something another chain.